include/boost/math/fft/algorithms.hpp

```diff
@@ -354,7 +355,7 @@
     // auto _1 = T{1};
     
     int nbits = 0;
-    std::vector<T> e2{e};
+    ::boost::container::static_vector<T,32> e2{e};
```

Comment:
The quickest way I found to get rid of heap memory here was to introduce the `static_vector`. I would have used an `std::array` but the class `T` doesn't necessarily have a default constructor.